jib-core/src/test/resources/sampleDockerfile

```diff
@@ -4,6 +4,7 @@ COPY libs /app/libs/
 COPY snapshot-libs /app/libs/
 COPY resources /app/resources/
 COPY classes /app/classes/
+COPY root /
```

Comment:
Just asking: does this open up the possibility that copying the files at `root/` might overwrite existing files, or is it that `root/` will never have a sub-directory called `app/`?

Reply:
Yes, it opens up that possibility.